uber/site_sections/registration.py

```diff
@@ -162,7 +162,6 @@ def delete(self, session, id, return_to='index?'):
                 ]}))
                 session.assign_badges(attendee.group, attendee.group.badges + 1, attendee.badge_type)
                 session.delete_from_group(attendee, attendee.group)
-                attendee.group.attendees.remove(attendee)
```

Comment:
This line already shows up in the ``session.delete_from_group()`` function which is called on the previous line.  Because it's already been called, this ``attendee`` no longer has a ``group`` set, which is what caused us to raise an ``AttributeError``.